tests/unit/services/opensearch/test_cluster_manager.py

```diff
@@ -30,12 +32,12 @@ def test_endpoint_strategy_path(self, monkeypatch, engine):
         endpoint = build_cluster_endpoint(
             DomainKey("my-domain", "us-east-1", get_aws_account_id()), engine_type=engine_type
         )
-        assert endpoint == f"localhost:4566/{engine_path_prefix}/us-east-1/my-domain"
+        assert endpoint == f"http://localhost:4566/{engine_path_prefix}/us-east-1/my-domain"
```

Comment:
You can not just change the content of the responses here. There might be clients which rely on the scheme not being present in the response.
Or is the scheme also contained in responses by AWS?

Reply:
As far as I understood it, these tests only test if our internal `build_custom_endpoint' method returns the correct answer. Since this method is only used in Opensearch, I thought it was within the scope of potential refactoring. Not sure if I am missing something though.